### PR TITLE
D8 un 835

### PR DESCRIPTION
**Pull request by soda736**

### CODE REVIEW

File: web/sites/nipolicingboard/themes/nipolicingboard_theme/templates/block/block--system-menu-block--menu-useful-links.html.twig [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * class, which still keeps it visible for screen-readers and assistive
+ * technology. Headings allow screen-reader and keyboard only users to navigate
+ * to or skip the links.
+ * See http://juicystudio.com/article/screen-readers-display-none.php and
+ * http://www.w3.org/TR/WCAG-TECHS/H42.html for more information.
+ */
+#}
+{% set heading_id = attributes.id ~ '-menu'|clean_id %}
+<div class="menu_wrapper">
+  <nav id="footer-menu" class="footer-menu block clearfix" aria-labelledby="{{ heading_id }}"{{ attributes|without('role', 'aria-labelledby', 'id') }}>
```
Comment (omahm) [inline web/sites/nipolicingboard/themes/nipolicingboard_theme/templates/block/block--system-menu-block--menu-useful-links.html.twig:36]: This ID doesn't make sense to me in the context of the twig file called 'menu-useful-links'. Is there a particular reason why it's called 'footer-menu' ?

**Review by omahm — COMMENTED** [review]
Looks good but just querying the useful links menu id

**Comment by soda736** [comment]
Good spot, I just copied and pasted this template, cheers.

**Review by omahm — APPROVED** [review]
LG 👍

**Pull request merged**